src/ActionKit/ActionTemplate/FileActionTemplate.php

```diff
@@ -0,0 +1,116 @@
+<?php
+namespace ActionKit\ActionTemplate;  
+use Exception;
+use ActionKit\Exception\UnableToWriteCacheException;
+use Twig_Loader_Filesystem;
+use Twig_Environment;
+
+/**
+ * Depends on Twig template engine
+ */
+
+class FileActionTemplate implements IActionTemplate
```

Comment:
And named the other as `CodeGenActionTemplate` since it's using `CodeGen` package to generate Actions